src/popup/router/components/InviteItem.vue

```diff
@@ -39,9 +71,10 @@ export default {
   },
   components: { Button, AmountSend, CopyIcon },
   filters: { formatDate },
-  data: () => ({ topUp: false, topUpAmount: 0, balance: 0 }),
+  data: () => ({ topUp: false, topUpAmount: 0, balance: 0, currencySigns }),
```

Comment:
This can be also extracted to a getter, maybe even to a global vuex getter, because it is used in several places.